config/finance/2024/Expenses.yml

```diff
@@ -107,16 +105,41 @@ September:
     Amount: '2000.00'
   - Category: JSON Schema Sponsorship
     Amount: '250.00'
+  - Category: AsyncAPI Conf on Tour 2024
+    Amount: '672.23'
 October:
   - Category: Bounty Program
     Amount: '1000.00'
   - Category: Swag Store
     Amount: '882.12'
   - Category: AsyncAPI Conf on Tour 2024
-    Amount: '962.01'
+    Amount: '289.78'
+  - Category: Community Manager
+    Amount: '2000.00'
+  - Category: Community Marketing Specialist
+    Amount: '2000.00'
+  - Category: JSON Schema Sponsorship
+    Amount: '250.00'
+November:
+  - Category: Swag Store
+    Amount: '472.36'
   - Category: Community Manager
     Amount: '2000.00'
   - Category: Community Marketing Specialist
     Amount: '2000.00'
   - Category: JSON Schema Sponsorship
     Amount: '250.00'
+December:
+  - Category: Bounty Program
+    Amount: '1400.00'
+  - Category: Swag Store
+    Amount: '917.99'
+  - Category: Community Manager
+    Amount: '2000.00'
+  - Category: Community Marketing Specialist
+    Amount: '2000.00'
+  - Category: JSON Schema Sponsorship
+    Amount: '250.00'
+  - Category: AsyncAPI Conf on Tour 2024
+    Amount: '2767.99'
+    
```

Comment:
@derberg we need a line addition here

Reply:
what do you mean? there is already one extra empty line 145